### PR TITLE
Older versions of ipset don't support comments

### PR DESCRIPTION
**Pull request by justinta**
This should also fix a problem where the module will report incorrectly if there
isn't a comment.

I have a feeling there is a better way to do this, so if anyone has a critique or a better way of doing this, I'm all ears :) 

Addresses this failing test: http://166.78.178.63:8080/job/branch_tests/job/2015.8/job/salt-2015_8-linode-debian8/lastFailedBuild/

### CODE REVIEW

**Comment by jtand** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Why did this get merged?! We just merged a commit in this module here: https://github.com/saltstack/salt/pull/30292

We need to double-check with @thegoodduke here and make sure we're all on the same page so we're not stomping on each other's changes.

**Comment by basepi** [comment]
@thegoodduke could you please review this one and the fixes in https://github.com/saltstack/salt/pull/30391 to make sure it's good? Some tests were failing which is why these changes were made, sorry we didn't check with you.

**Comment by jtand** [comment]
@cachedout @thegoodduke, my bad I should have referenced #30292 and pinged him. This was after his change to address the failing test from it. We didn't pick it up because it was buried in the other failing tests at the time.

File: salt/modules/ipset.py [hunk 1]
```diff
@@ -439,22 +444,31 @@ def check(set=None, entry=None, family='ipv4'):
                                                              ipaddress.ip_address(end))
                 entries = []
                 for network in networks:
+                    entries.append(network.with_prefixlen)
```
Comment (thegoodduke) [inline salt/modules/ipset.py]: what for here?  Can you explain it, please.

Comment (jtand) [inline salt/modules/ipset.py]: I took that line out, I didn't mean to put it in. It was just a remnant from testing.

**Pull request merged**